azext_iot/tests/test_iot_ext_int.py

```diff
@@ -629,7 +629,7 @@ def test_hub_modules(self):
                          self.exists("[?moduleId=='$edgeHub']")])
 
         self.cmd('iot hub module-identity list -d {} -n {} -g {} --top -1'.format(edge_device_ids[0], LIVE_HUB, LIVE_RG),
-                 checks=[self.check('length([*])', 4),
+                 checks=[self.check('length([*])', 3),
```

Comment:
I think this should still be 4, but without `--top -1` in the command.

Since we're using `top` as a direct index now, a value of `[:-1]` is valid, as it just removes the last item.

Reply:
This would be 3, the command is the same as just previous command except for "Top" parameter. confirmed with the execution of integration test

Reply:
Oh, I see now - in that case it seems like a duplicate / strange test - would it make more sense to use a positive value for `top` and verify that you only get [`top`] results?

Reply:
It would be the same case in that scenario too, top with +ve value will check [top] results and -ve top will result in (total result - top) = result